packages/core-contracts/contracts/interfaces/IERC721Metadata.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IERC165.sol";
+
+interface IERC721Metadata is IERC165 {
```

Comment:
do we really need the metadata? My guess is that we need basic functionality for governance.